tests/appsec/waf/test_addresses.py

```diff
@@ -338,7 +338,8 @@ def test_security_scanner(self):
         interfaces.library.assert_waf_attack(r, pattern="appscan_fingerprint", address="server.request.path_params")
 
 
-@released(golang="1.36.0", dotnet="?", java="?", nodejs="?", php_appsec="?", python="?", ruby="?")
+@released(golang="1.36.0", dotnet="?", java="0.96.0", nodejs="?", php_appsec="?", python="?", ruby="?")
+@irrelevant(context.library == "java" and context.weblog_variant != "spring-boot")
```

Comment:
```suggestion
@released(golang="1.36.0", dotnet="?", java={"spring-boot": "0.96.0"}, nodejs="?", php_appsec="?", python="?", ruby="?")
```
I believe, new version syntax should here.
@cbeauchesne please, confirm

Reply:
Hi @ValentinZakharov 

Yes it does. But what you dit was basically : *Spring-boot is ok starting  `0.96.0`, everything else is ok for all tracer version*. What you want is probably `{"spring-boot": "0.96.0", "*": "?"}`.

Though, the doc is not up to date. On it.

Reply:
Actually, it's up to date :) 

https://github.com/DataDog/system-tests/blob/main/docs/edit/declare-versions.md